src/domain/wallet-invoices/index.types.d.ts

```diff
@@ -3,13 +3,20 @@ type WalletInvoice = {
   readonly walletId: WalletId
   readonly selfGenerated: boolean
   readonly pubkey: Pubkey
+  readonly fiat: FiatAmount | null
+  readonly currency: WalletCurrency
   paid: boolean
 }
 
 type WalletInvoiceValidator = {
   validateToSend(fromWalletId: WalletId): true | ApplicationError
 }
 
+type WalletFactoryArgs = {
```

Comment:
This should be `WalletInvoiceFactoryConfig` I think.
We haven't made it explicit yet but I have been following the pattern `Config` for constructors and `Args` for functions on types.

Reply:
not sure this is what I have seen elsewhere. @vindard what do you think?

Reply:
Juan and I have been back and forth on this in the past as well. I think framing the top level function args as `..Config` and the inner function args as `...Args` makes sense and helps to separate the abstractions better actually 🤔

Reply:
yep should have Config suffix